boards/arc/hsdk4xd/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2023 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_HSDK4XD
+
+config BOARD
+	default "hsdk4xd"
+
+# TODO
```

Comment:
Do we really need that empty `# TODO` tag here?

Reply:
Support of HSDK4xD in minimal configuration announced in this PR. Kconfig supposed to be extended with additional board features. ``` TODO ```  hints about that =)

Reply:
That's good, but still it worth saying what need to be done, as in 2 weeks we'll all forget about that.

Reply:
or even better, instead of this 'TODO' in the code, create a Github issue for it.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/57430